### PR TITLE
Added Null Search Results Catch

### PR DESCRIPTION
**Pull request by rohitmusti**
Previously, a search with no results would merely end up empty. I added an if else statement to return this output.

No results were found for "searchTerm". Please try another search, or let us know if something is missing.

### CODE REVIEW

**Comment by rdebeasi** [comment]
Resolves #125 and part of #121

File: config.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # https://gohugo.io/getting-started/configuration/
 # https://github.com/MunifTanjim/minimo/blob/master/exampleSite/config.toml
 
-baseURL = "https://rht-labs.github.io/practice-library/"
-#baseURL = "https://rohitmusti.github.io/practice-library/"
+#baseURL = "https://rht-labs.github.io/practice-library/"
+baseURL = "https://rohitmusti.github.io/practice-library/"
```
Comment (tenfourty) [inline config.toml]: this looks like a mistake.

File: docs/404.html [hunk 2]
```diff
@@ -12,15 +12,15 @@
...
 
-<meta name="generator" content="Hugo 0.43" />
+<meta name="generator" content="Hugo 0.44" />
 
   <title>404 Page not found • Open Practice Library</title>
   <link rel='canonical' href=''>
   
   
-  <link rel='icon' href='https://rht-labs.github.io/practice-library/favicon.ico'>
+  <link rel='icon' href='https://rohitmusti.github.io/practice-library/favicon.ico'>
```
Comment (tenfourty) [inline docs/404.html]: lots of hard coded links here that have been changed to the wrong url.

**Comment by tenfourty** [comment]
why are we checking html files into this repo - shouldn't all of these be generated by hugo?

Also all urls should be relative and the baseurl should be set in the config.toml file - this PR highlights many places where there are hardcoded links that should be changed to relative ones, some comments just to highlight the issue from me.

**Comment by tenfourty** [comment]
on reviewing this - it looks like your .gitignore is not working because the docs dir shouldn't be added to this repo.

**Comment by rohitmusti** [comment]
Closing this PR and re-opening another one under another branch! This is just b/c I am working on some new features and want this PR to live in its own branch so that I can keep my master up-to-date with my last working local code. I have written down all of these issues and am addressing them in the separate PR

Thank you for your comments @tenfourty @rdebeasi

**Pull request closed without merging**